src/screens/AccountPin.js

```diff
@@ -120,7 +121,7 @@ class AccountPinView extends React.PureComponent {
 	render() {
 		const title = 'ACCOUNT PIN';
 		return (
-			<View style={styles.body}>
+			<KeyboardScrollView style={styles.body} extraHeight={120}>
```

Comment:
probably not a big deal but does this hardcoded `extraHeight` look okay with smaller devices?